### DISCUSSION TITLE
Why is accessibility services needed?

### DISCUSSION THREAD

**Discussion by BenPyton** [otakuhqz/Omega#448] [Q&A]
Hi, 
I love this launcher so far!

However, there is a little thing that annoys me: when I'm using the back button on the home screen, it brings me to the Accessibility settings of the phone.

I can enable this service for Neo Launcher to avoid that behaviour, but I don't know why it is requesting that, there is no description provided by the application.

Can someone explains to me why the launcher needs this service?
It could be great to add a description too. 
For example, Bitwarden provides a detailed description saying it needs accessibility to detect and autofill password fields on other apps.

Thanks all.

**Comment by ltguillaume** [comment]
You might have set the back gesture to "Sleep", which requires the permission to run an accessibility service.

**Reply by BenPyton** [reply]
I've checked, but no I didn't. The back button is set to "Back Button" and nothing is set to "Sleep" (see screenshot).
![image](https://github.com/user-attachments/assets/5066a7f0-5049-4fc1-b539-5fc2513215ca)

**Reply by ltguillaume** [reply]
Set it to Do Nothing, otherwise it will try to emulate the behavior of the Back button, which also requires an Accessibility service.
